src/tests/unit/tests/background/issue-details-text-generator.test.ts

```diff
@@ -20,19 +20,23 @@ describe('Issue details text builder', () => {
 
     beforeEach(() => {
         sampleIssueDetailsData = {
-            pageTitle: 'pageTitle<x>',
-            pageUrl: 'pageUrl',
-            ruleResult: {
-                failureSummary: 'RR-failureSummary',
-                guidanceLinks: [{ text: wcagTags[0] }, { text: wcagTags[1] }],
-                help: 'RR-help',
-                html: 'RR-html',
-                ruleId: 'RR-rule-id',
-                helpUrl: 'RR-help-url',
-                selector,
-                snippet: 'RR-snippet   space',
-            } as any,
-        };
+            rule: {
+                description: 'RR-help',
```

Comment:
The values were probably kept the same to avoid updating the snapshot, but should we change the values to more closely match the new structure of the data? It might be confusing when looking at this test in the future why it says "RR" when it's not mapped to a `ruleResult` or "failureSummary" when the key is a `howToFixSummary`.

Reply:
Good point! I think we should do it in a different PR. If the snapshot doesn't change in this PR it helps show that the behavior has not changed?